third_party/dialog/DialogSDK/interfaces/ftdf/src/ad_ftdf_phy_api.c

```diff
@@ -158,8 +158,8 @@ void ad_ftdf_init_phy_api(void)
     NVIC_ClearPendingIRQ(FTDF_WAKEUP_IRQn);
     NVIC_EnableIRQ(FTDF_WAKEUP_IRQn);
 
-    NVIC_ClearPendingIRQ(FTDF_GEN_IRQn);
-    NVIC_EnableIRQ(FTDF_GEN_IRQn);
+    //NVIC_ClearPendingIRQ(FTDF_GEN_IRQn);    // Disable ftdf interrupts
```

Comment:
Generally we try not to modify third party code.  Is there a way that we can do this in the platform code instead of in the SDK?